scriptworker/constants.py

```diff
@@ -402,27 +425,35 @@
                     '/mozilla-mobile/reference-browser',
                 ),
             }),
+            'application-services': frozendict({
+                'application-services-repo': (
+                    '/mozilla/application-services',
+                )
+            }),
         }),
     },
     'prebuilt_docker_image_task_types': {
         'by-cot-product': frozendict({
             'firefox': ('decision', 'action', 'docker-image'),
             'thunderbird': ('decision', 'action', 'docker-image'),
             'mobile': 'any',  # all allowed
+            'application-services': 'any',  # all allowed
         }),
     },
     'source_env_prefix': {
         'by-cot-product': frozendict({
             'firefox': 'GECKO',
             'thunderbird': 'COMM',
             'mobile': 'MOBILE',
+            'application-services': 'APPSERVICES',
```

Comment:
I wonder if `APPSERVICES` is too narrow. Do you know what kind of projects would fall under this `cot_product`?